src/applications/mhv/medications/containers/Prescriptions.jsx

```diff
@@ -27,13 +27,15 @@ const Prescriptions = () => {
 
   const sortRxList = useCallback(
     () => {
-      const newList = [...prescriptions];
-      newList.sort(a => {
-        return a.refillStatus.toLowerCase() === sortOption.toLowerCase()
-          ? -1
-          : 0;
-      });
-      dispatch(setSortedRxList(newList));
+      if (sortOption) {
+        const newList = [...prescriptions];
```

Comment:
I know unit tests will go in a separate PR, it would be good to split this logic into a separate function and get it unit tested there :)

Reply:
aah Im sorry Adrian I just merged it and missed this comment.

Reply:
@robertbylight nope all good, I just meant to handle in the testing PR 🙂